tothic/tothic.go

```diff
@@ -39,6 +41,13 @@ type PathParam struct {
 	Provider string `json:"provider"`
 }
 
+func (p PathParam) DBID() apidef.ObjectId {
```

Comment:
I do get why profile object needs to have it but why PathParam?

Reply:
The `SetKey` func is called with PathParam in the code so it mockly implements it to make the code compile